README.org

```diff
@@ -118,6 +128,35 @@ In short, =brew= doesn't really support options. They break time to time ([[http
   $ brew install emacs-plus [options]
 #+END_SRC
 
+** Emacs 30
+
+*** Included
+
+By default =emacs-plus@30= uses the following features.
+
+- Injected =PATH= value from user shell (see separate section explaining this feature).
+- Cocoa version, e.g. builds =Emacs.app=.
+- Unconditional support for =gnutls=, =librsvg=, =libxml2=, =little-cms2= and dynamic modules.
+
+*** Options
+
+| Option                  | Description                                                                  |
+|-------------------------+------------------------------------------------------------------------------|
```

Comment:
not a big deal, but you removed extra `-` and spaces by ignoring `=` characters

Reply:
Ah, I should have noticed that. I hide markers so that impacts that, but I need to turn that off when working in other peoples tables.